litiengine/src/test/java/de/gurkenlabs/litiengine/util/ColorHelperTests.java

```diff
@@ -40,44 +41,34 @@ private static Stream<Arguments> getColorFromHexString() {
         Arguments.of("#0000ff", Color.BLUE));
   }
 
-  @Test
-  void testRedFromAlphaHexString() {
-    String red200 = "#c8ff0000";
-
-    Color redDecoded = ColorHelper.decode(red200);
-
-    Color alphaRed = new Color(255, 0, 0, 200);
-    assertEquals(alphaRed.getRed(), redDecoded.getRed());
-  }
-
-  @Test
-  void testGreenFromAlphaHexString() {
-    String red200 = "#c8ff0000";
+  @Nested
+  class DecodeTest
+  {
+      String red200 = "#c8ff0000";
```

Comment:
The blue, green, and alpha in this color should tested with a non-zero value, just to be sure it's actually being initialized correctly.

For example `c8ffddee`